.well-known/apple-app-site-association.json

```diff
@@ -0,0 +1,19 @@
+{
+  "applinks": {
+    "apps": [],
+    "details": [
+      {
+        "appID": "3YE4W86L3G.Tavultesoft.Keyman",
+        "paths": ["/keyboards/install/*"],
```

Comment:
```suggestion
        "paths": ["/keyboards/install/*"]
```

Reply:
Sorry, forgot about that bit.  Also, I'm not so familiar with doing multi-line suggestions.